src/apis/NetInfo.md

```diff
@@ -6,6 +6,6 @@ wip: true
 
 NetInfo has been removed from React Native.
 
-Please use [bindings](https://github.com/reason-react-native/netinfo) to the
-replacement
+======= Please use [bindings](https://github.com/reason-react-native/netinfo) to
```

Comment:
I suppose this is another editor artifact.

Please revise as 
`Please refer to [bindings](https://github.com/reason-react-native/netinfo) for the replacement [community package](https://github.com/react-native-community/react-native-netinfo).`